### PR TITLE
[PE-3898] Improve and make consistent input number markup versions

### PR DESCRIPTION
**Pull request by jllr**
https://ctl.atlassian.net/browse/PE-3898

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/495/ for more details.

**Comment by jllr** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/496/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/497/ for more details.

File: src/website/views/components/forms/number-input.pug [hunk 1]
```diff
@@ -56,119 +104,189 @@ p.-text
...
+            </i>
+          </div>
+        </button>
       </div>
     </div>
 
 h2 Expanded version
 p.-text
-  | There is available an expanded version of the number input component.
+  | Apply the <code>-alt</code> class to render alternative number input version.
```
Comment (mattnickles) [inline src/website/views/components/forms/number-input.pug]: The custom element is still using the attribute `expanded`. Can we unify the two? We'll need something more descriptive than `alt` incase we introduce another variant.

Comment (jllr) [reply src/website/views/components/forms/number-input.pug]: @mattnickles makes sense. What about -inline? Any other suggestion?

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/528/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/534/ for more details.

**Comment by jllr** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/568/ for more details.

**Comment by jllr** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/571/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/579/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/585/ for more details.

**Comment by mattnickles** [comment]
@jllr can we close this PR as the latest number input changes are in #541 ?

**Pull request closed without merging**